ansible/roles/ceph-release/tasks/push.yml

```diff
@@ -41,21 +41,22 @@
   args:
     chdir: ceph
 
-- name: "git checkout {{ branch }}"
-  command: git checkout upstream/{{ branch }}
-  args:
-    chdir: ceph
-
-# In case any commits got pushed to {{ branch }} while we were building
-- name: "merge {{ branch }}-release changes back into {{ branch }}"
-  command: git merge -m "Merging {{ branch }}-release {{ version }} commit into {{ branch }}" {{ branch }}-release
-  args:
-    chdir: ceph
-  
-- name: "push version commit to {{ branch }}"
-  command: git push upstream {{ branch }}
-  args:
-    chdir: ceph
+- name: "create pull request to merge {{ branch }}-release back into {{ branch }}"
+  uri:
+    url: https://api.github.com/repos/ceph/ceph/pulls
+    method: POST
+    status_code: 201
+    headers:
+      Accept: "application/vnd.github.v3+json"
+      Authorization: "token {{ token }}"
+    body:
+      title: "v{{ version }}"
+      body: "{{ pr_checklist }}"
```

Comment:
It'd be better to just read the file `.github/pull_request_template.md` before this task, no?

Reply:
Read it and then programmatically check the boxes? Is that what you're suggesting? I suppose.

Reply:
Actually, even if the checklist changes, I'd still have to go in this playbook and change which boxes get checked so it's kind of a moot point.

Reply:
@batrick can we merge this as-is now?

Reply:
ya